projects/igniteui-angular/src/lib/query-builder/query-builder-tree.component.ts

```diff
@@ -519,11 +522,12 @@ export class IgxQueryBuilderTreeComponent implements AfterViewInit, OnDestroy {
     }
 
     constructor(public cdr: ChangeDetectorRef,
+        public dragService: IgxQueryBuilderDragService,
```

Comment:
Do we need the service public?

Reply:
I still think we should not expose it publicly. Maybe make it hidden.

Reply:
The entire IgxQueryBuilderTreeComponent is marked as hidden:
![image](https://github.com/user-attachments/assets/e20cad48-f4eb-4a7a-b71f-37241623ff33)